### ISSUE TITLE
Prefer basic name for default and preferred name (not disambiguation name)

### ISSUE THREAD

**Issue by nvkelso** [whosonfirst-data/whosonfirst-data#885]
It looks like we've inherited disambiguation names for default `wof:name` and name:*_x_preferred` when we should prefer the basic name instead.

A disambiguation name is useful for saying `Portland (Oregon)` versus `Portland (Maine)`, but the basic name for each is `Portland` and our wof:id and place hierarchy should establish the disambiguation. 

Some people prefer disambiguation names, especially when viewing place names in a list so we should preserve these names probably in a new `name:*_x_disambiguation` slot?

This is especially common in Germany but happens world wide.

**Two examples** from https://github.com/whosonfirst-data/whosonfirst-data/pull/884/files:

- data/101/748/705/101748705.geojson
    - both of these should be just **`Freiburg`**
    - `"wof:name":"Freiburg I'm Breisgau",`
    - `"name:deu_x_preferred":["Freiburg im Breisgau" ],`
- data/101/748/705/101748705.geojson
    - both of these should be just **`Halle `**
    - `"wof:name":"Halle (Saale)",`
    - `"name:deu_x_variant":[ "Halle" ],`

**Comment by thisisaaronland** [comment]
_goes to get a cup of coffee..._

**Comment by thisisaaronland** [comment]
"Who's On First prefers map display over lists..." <-- For the record, this is not true but also not the question at hand.

I don't have any problem in principle adding `_x_disambiguation` names but this seems like the purpose and the reason for the `label:` fields.

Specifically: Labels, like `wof:name` are singletons (unless someone can remind me of the compelling reason why they _should_ be lists) and `name:` properties are not. Would there be a case where a disambiguated name would have multiple options? How would you... disambiguate them?

Then there is the larger philosophical question about whether or not the act of disambiguation transform a name in to a label...

**Comment by nvkelso** [comment]
Remind me about `wof:label`? I see it listed here but no where else in docs:

- https://github.com/whosonfirst/whosonfirst-properties/blob/9ef52e6a10e373dff7157d62058c68d8f68c7d28/aliases/property_aliases.json#L193

What's an example? We've been using `mz:hierarchy_label` quite a bit, but that's a boolean.

- https://github.com/whosonfirst/whosonfirst-properties/blob/master/properties/mz.md#hierarchy_label

_transmutation of names into labels..._ :cloud_with_lightning: ⚡️

**Comment by thisisaaronland** [comment]
Well, both `wof:label` and `label:{LANG}_x_{DETAIL}`.

These are meant to be like `wof:name` and `name:{LANG}_x_{DETAIL}` but for fully qualified names, like an address or a disambiguated name.

We've discussed it over the years mostly in the context of Pelias (for example `Montreal QC Canada` rather than just `Montreal`) but it's just never made the cut, relative to all the other things.

**Comment by nvkelso** [comment]
This needs to be squared with discussion in https://github.com/whosonfirst-data/whosonfirst-data/pull/823#issuecomment-316213254.

Some related Pelias links after chatting with @dianashk:

- use of `mz:hierarchy_label`: https://github.com/pelias/wof-admin-lookup/blob/0ba80b9a3a1c597d53828753252006404967eed3/src/pip/components/filterOutUnimportantRecords.js
- `wof:label`: https://github.com/pelias/wof-admin-lookup/blob/ff5db5d87031d12e45a090eb98bd43d0e614f85e/src/pip/components/getDefaultName.js#L19-L21
- `wof:label` during actual localization (when someone asks for labels back in their language): https://github.com/pelias/placeholder/blob/1a6fe01df8efe34a57b2bf50f08b694933e89c10/prototype/wof.js#L86

Short version is there some conflicting usages here and we need to standardize, probably Pelias code needs an update, too. But I'm going on vacation so will need to wait ;)

**Comment by missinglink** [comment]
Heya,

There are definately pros/cons to the "Display vs. Raw Data" discussion, I personally feel like having data in it's rawest form is the most 'generally useful' and so it's what we try to do for search.

Having said that, we also provide a 'display' property called 'label'

eg:
```
      "properties": {
        ...
        "country_a": "AUS",
        "region": "Victoria",
        "county": "Port Phillip (C)",
        "localadmin": "St Kilda (Vic.)",
        "locality": "St Kilda",
        "neighbourhood": "St Kilda South",
        ...
        "label": "Inkerman Street, St Kilda, Australia"
      },
```

You can see the label we provide is `Inkerman Street, St Kilda, Australia`, there is a bunch of business logic in place to try and select relevant-yet-short labels for display to our end users.

For the most part this is fine for display, however some users would like to format their labels differently and so having the data stemmed and normalised allows them to recompose the label however is best for their own domain.

In the example above, choosing to format as `{{localadmin}} {{region}} {{country_a}}` would produce the label `St Kilda (Vic.) Victoria AUS` which contains duplicate state information.

The other consideration for us is at index time, when we ingest the tokens we would prefer to use tokens from a property which doesn't contain additional punctuation and abbreviations, this helps reduce the index size and (more importantly) improves full-text matching accuracy.

**Comment by nvkelso** [comment]
Per IRL discussion as captured in PR https://github.com/whosonfirst/whosonfirst-properties/pull/35:

We're going to use `label:{lang}_x_preferred_disambiguation` for this (and `label:{lang}_x_variant_disambiguation`).

**Comment by nvkelso** [comment]
Another example: 

data/102/063/191/102063191.geojson

```
"name:eng_x_preferred":[
        "Lindau (Bodensee) "
    ],
```

**Comment by nvkelso** [comment]
See also Australia names.
